mephisto/abstractions/databases/README.md

```diff
@@ -1,10 +1,10 @@
 # MephistoDB implementations
-This folder contains implementations of the `MephistoDB` abstraction. Databases can currently be configured using the `mephisto.database._database_type` flag, though all databases other than `local` are considered to be in BETA at this point.
+This folder contains implementations of the `MephistoDB` abstraction. Databases can currently be configured using the `mephisto.database._database_type` flag.
 
 ## `LocalMephistoDB`
-An implementation of the Mephisto Data Model outlined in `MephistoDB`. This database stores all of the information locally via SQLite. Some helper functions are included to make the implementation cleaner by abstracting away SQLite error parsing and string formatting, however it's pretty straightforward from the requirements of MephistoDB.
+Activated with `mephisto.database._database_type=local`. An implementation of the Mephisto Data Model outlined in `MephistoDB`. This database stores all of the information locally via SQLite. Some helper functions are included to make the implementation cleaner by abstracting away SQLite error parsing and string formatting, however it's pretty straightforward from the requirements of MephistoDB.
```

Comment:
Just curious, if `_database_type` is a public facing prop, why is it `_` prefixed? Does the prefix denote something else?

Reply:
In this case it has to do with the fact that databases haven't been raised to the same level of usage as the other abstractions. Eventually you'll specify `mephisto.database=local` and that will auto-load the default arguments for a `LocalDatabase`, then you can override with `mephisto.database.<arg>=<val>`, but I haven't configured any of that yet for databases. This setup however is future-proof, as the underscore type parameter will always be present and usable as an override.